wagtail/models/__init__.py

```diff
@@ -3647,24 +3647,37 @@ def start(self, workflow_state, user=None):
 
         return super().start(workflow_state, user=user)
 
+    def _user_in_groups(self, user):
+        # Cache the check whether "this user is in any of this
+        # GroupApprovalTask's groups" on the user object, in case we do it
+        # against the same user and task multiple times in a request.
+        # Use a dict to map the task id to the check result, in case we also
+        # check against different GroupApprovalTasks for the same user.
+        attr = "_group_approval_task_checks"
```

Comment:
Issue: `attr` is only used once here. It might be better to either inline it into the below method, or use it in the subsequent times we'd use it (ideally the latter)

Reply:
Thanks, I changed this to use the walrus operator instead.